### PR TITLE
Fix rpmlint errors

### LINKED ISSUE #2236: Rpmlint warning
### Describe the bug

Currently the rpmlint complains about a number of things which can be solved with some adjustments.

### To Reproduce

1. Go to https://build.opensuse.org/package/show/home:SchoolGuy:branches:systemsmanagement/cobbler
2. Checkout the project or go to the buildlog
3. Scroll down to the rpmlint report
4. See the warnings

### Expected behavior

Less warnings

### Version

3.1.1.

### Additional context

````
[   17s] RPMLINT report:
[   17s] ===============
[   18s] cobbler.noarch: W: files-duplicate /etc/cobbler/boot_loader_conf/bootcfg_esxi5.template /etc/cobbler/boot_loader_conf/bootcfg_esxi51.template
[   18s] cobbler.noarch: W: files-duplicate /etc/cobbler/boot_loader_conf/bootcfg_esxi60.template /etc/cobbler/boot_loader_conf/bootcfg_esxi55.template:/etc/cobbler/boot_loader_conf/bootcfg_esxi65.template
[   18s] cobbler.noarch: W: files-duplicate /etc/cobbler/boot_loader_conf/gpxe_system_esxi6.template /etc/cobbler/boot_loader_conf/gpxe_system_esxi5.template
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/__pycache__/__init__.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/__pycache__/__init__.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/__pycache__/cexceptions.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/__pycache__/cexceptions.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/__pycache__/field_info.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/__pycache__/field_info.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/actions/__pycache__/__init__.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/actions/__pycache__/__init__.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/cobbler_collections/__pycache__/__init__.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/cobbler_collections/__pycache__/__init__.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/items/__pycache__/__init__.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/items/__pycache__/__init__.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/__pycache__/__init__.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/modules/__pycache__/__init__.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/authentication/__pycache__/__init__.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/modules/authentication/__pycache__/__init__.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/authorization/__pycache__/__init__.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/modules/authorization/__pycache__/__init__.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/authorization/__pycache__/allowall.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/modules/authorization/__pycache__/allowall.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/installation/__pycache__/__init__.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/modules/installation/__pycache__/__init__.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/managers/__pycache__/__init__.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/modules/managers/__pycache__/__init__.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/nsupdate_delete_system_pre.py /usr/lib/python3.7/site-packages/cobbler/modules/nsupdate_add_system_post.py
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/modules/serializers/__pycache__/__init__.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/modules/serializers/__pycache__/__init__.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/web/__pycache__/__init__.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/web/__pycache__/__init__.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/web/__pycache__/settings.cpython-37.pyc /usr/lib/python3.7/site-packages/cobbler/web/__pycache__/settings.cpython-37.opt-1.pyc
[   18s] cobbler.noarch: W: files-duplicate /usr/lib/python3.7/site-packages/cobbler/web/templatetags/__pycache__/__init__.cpython-37.opt-1.pyc /usr/lib/python3.7/site-packages/cobbler/web/templatetags/__pycache__/__init__.cpython-37.pyc
[   18s] cobbler.noarch: W: files-duplicate /var/lib/cobbler/templates/esxi5-ks.cfg /var/lib/cobbler/templates/esxi4-ks.cfg
[   18s] cobbler.src:117: W: hardcoded-packager-tag Cobbler Developers <cobbler@lists.fedorahosted.org>
[   18s] The Packager tag is hardcoded in your spec file. It should be removed, so as
[   18s] to use rebuilder's own defaults.
[   18s] 
[   18s] cobbler.noarch: W: incoherent-logrotate-file /etc/logrotate.d/cobblerd
[   18s] Your logrotate file should be named /etc/logrotate.d/<package name>.
[   18s] 
[   18s] cobbler.noarch: W: missing-dependency-to-logrotate for logrotate script /etc/logrotate.d/cobblerd
[   18s] This package installs a file in /etc/logrotate.d/ but doesn't require
[   18s] logrotate to be installed. Because logrotate is not part of the essential
[   18s] packages, your package should explicitely depend on logrotate to make sure
[   18s] that your logrotate job is executed. If it is an optional feature of your
[   18s] package, recommend or suggest logrotate.
[   18s] 
[   18s] cobbler.noarch: W: non-executable-script /usr/lib/python3.7/site-packages/cobbler/web/manage.py 644 /usr/bin/python3 
[   18s] cobbler.noarch: W: non-executable-script /usr/share/bash-completion/completions/cobbler 644 /usr/bin/env bash
[   18s] cobbler.noarch: W: non-executable-script /usr/share/cobbler/bin/settings-migration-v1-to-v2.sh 644 /bin/bash 
[   18s] This text file contains a shebang or is located in a path dedicated for
[   18s] executables, but lacks the executable bits and cannot thus be executed.  If
[   18s] the file is meant to be an executable script, add the executable bits,
[   18s] otherwise remove the shebang or move the file elsewhere.
[   18s] 
[   18s] cobbler.noarch: W: percent-in-%pre
[   18s] The %pre scriptlet contains a "%" in a context which might indicate it being
[   18s] fallout from an rpm macro/variable which was not expanded during build.
[   18s] Investigate whether this is the case and fix if appropriate.
[   18s] 
[   18s] cobbler.noarch: W: script-without-shebang /srv/www/cobbler/svc/services.py
[   18s] This text file has executable bits set or is located in a path dedicated for
[   18s] executables, but lacks a shebang and cannot thus be executed.  If the file is
[   18s] meant to be an executable script, add the shebang, otherwise remove the
[   18s] executable bits or move the file elsewhere.
[   18s] 
[   18s] cobbler.noarch: W: self-obsoletion cobbler-nsupdate < 3.0.99 obsoletes cobbler-nsupdate = v3.1.1-0
[   18s] The package obsoletes itself.  This is known to cause errors in various tools
[   18s] and should thus be avoided, usually by using appropriately versioned Obsoletes
[   18s] and/or Provides and avoiding unversioned ones.
[   18s] 
[   18s] cobbler.noarch: W: suse-missing-rclink cobblerd
[   18s] The package contains an init script or systemd service file but lacks the
[   18s] symlink /usr/sbin/rcFOO -> /usr/sbin/service
[   18s] 
[   18s] cobbler.noarch: W: systemd-service-without-service_add_post cobblerd.service
[   18s] The package contains a systemd service but doesn't contain a %post with a call
[   18s] to service_add_post.
[   18s] 
[   18s] cobbler.noarch: W: systemd-service-without-service_add_pre cobblerd.service
[   18s] The package contains a systemd service but doesn't contain a %pre with a call
[   18s] to service_add_pre.
[   18s] 
[   18s] 3 packages and 0 specfiles checked; 0 errors, 33 warnings.
````

### PR DESCRIPTION
**Pull request by SchoolGuy**
This should fix #2236 and improve the packaging of cobbler.

### CODE REVIEW

File: cobbler.spec [hunk 1]
```diff
@@ -264,16 +266,23 @@ mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,
...
+
 
 %pre
 %if %{_vendor} == "debbuild"
 if [ "$1" = "upgrade" ]; then
 %else
 if [ $1 -ge 2 ]; then
+    %if 0%{?suse_version}
+    %service_add_pre cobblerd.service
+    %endif
```
Comment (Conan-Kudo) [inline cobbler.spec]: This doesn't do anything useful, we don't have SysV scripts to migrate from and haven't had them for a long time.

File: cobbler.spec [hunk 2]
```diff
@@ -305,12 +314,21 @@ fi
 %else
 %post
 %systemd_post cobblerd.service
+%if 0%{?suse_version}
+%service_add_post cobblerd.service
+%endif
```
Comment (Conan-Kudo) [inline cobbler.spec]: This is redundant, as `%systemd_post` does the same thing.

File: cobbler.spec [hunk 3]
```diff
@@ -305,12 +314,21 @@ fi
...
 
 %preun
 %systemd_preun cobblerd.service
+%if 0%{?suse_version}
+%service_del_preun cobblerd.service
+%endif
```
Comment (Conan-Kudo) [inline cobbler.spec]: This is redundant, as `%systemd_preun` does the same thing.

File: cobbler.spec [hunk 4]
```diff
@@ -305,12 +314,21 @@ fi
...
 
 %postun
 %systemd_postun_with_restart cobblerd.service
+%if 0%{?suse_version}
+%service_del_postun cobblerd.service
+%endif
```
Comment (Conan-Kudo) [inline cobbler.spec]: This is redundant, as `%systemd_postun` does the same thing.

File: cobbler.spec [hunk 5]
```diff
@@ -264,16 +266,23 @@ mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,
 # systemd
 mkdir -p %{buildroot}%{_unitdir}
 mv %{buildroot}%{_sysconfdir}/cobbler/cobblerd.service %{buildroot}%{_unitdir}
+ln -sf service %{buildroot}%{_sbindir}/rccobblerd
 
 # cobbler-web
 rm %{buildroot}%{_sysconfdir}/cobbler/cobbler_web.conf
 
+# Remove duplicates
+%fdupes -s %{buildroot}%{_prefix}
```
Comment (Conan-Kudo) [inline cobbler.spec]: This will fail on Debian/Ubuntu (but not Fedora/CentOS). This is also strongly ill-advised, as you are running deduplication across all of `/usr`, which can have bad side-effects.

File: cobbler.spec [hunk 6]
```diff
@@ -340,6 +358,7 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %{python3_sitelib}/cobbler/
 %{python3_sitelib}/cobbler-*
 %{_unitdir}/cobblerd.service
+%{_sbindir}/rccobblerd
```
Comment (Conan-Kudo) [inline cobbler.spec]: This will _only_ work on SUSE distributions, so this needs to be conditioned out.

File: cobbler.spec [hunk 7]
```diff
@@ -264,16 +266,23 @@ mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,
 # systemd
 mkdir -p %{buildroot}%{_unitdir}
 mv %{buildroot}%{_sysconfdir}/cobbler/cobblerd.service %{buildroot}%{_unitdir}
+ln -sf service %{buildroot}%{_sbindir}/rccobblerd
```
Comment (Conan-Kudo) [inline cobbler.spec]: This will _only_ work on SUSE distributions, so don't create the symlink for anything else. You'll just create a broken program invocation.

**Comment by Conan-Kudo** [comment]
@SchoolGuy The rpmlintrc file does not belong here, strictly speaking. But, if you're going to have it, it needs to be called `cobbler.rpmlintrc`. That name works across all distributions using rpmlint.

File: cobbler.spec (hunk 1 above)
Comment (SchoolGuy) [reply cobbler.spec]: Correct me if I am wrong but if I understood this wiki article correctly it does more than that: https://en.opensuse.org/openSUSE:Systemd_packaging_guidelines#Register_systemd_unit_files_in_install_scripts

Comment (SchoolGuy) [reply cobbler.spec]: Hm okay. Then I will revert this. The linter complained and now doesn't so I will add this to the rpmlintrc along with the service calls below.

File: cobbler.spec (hunk 5 above)
Comment (SchoolGuy) [reply cobbler.spec]: I have removed fdupes and instead will ignore the warning.

File: cobbler.spec (hunk 6 above)
Comment (SchoolGuy) [reply cobbler.spec]: Done

File: cobbler.spec (hunk 7 above)
Comment (SchoolGuy) [reply cobbler.spec]: Done

File: cobbler.spec [hunk 8]
```diff
@@ -36,7 +36,7 @@
 %define apache_user www-data
 %define apache_group www-data
 %define apache_log /var/log/apache2
-%define apache_webconfigdir /etc/apache2/conf.d
+%define apache_webconfigdir /etc/apache2/conf-available
```
Comment (Conan-Kudo) [inline cobbler.spec]: I originally had `conf-available` but had to switch to `conf.d` because the `setup.py` did not set this up correctly. The logic there for the destination for the apache configs needs to be fixed.

Comment (SchoolGuy) [reply cobbler.spec]: Apparently Ubuntu only builds with errors and Debian doesn't build at all. Could you please have a look? https://build.opensuse.org/package/show/home:SchoolGuy/cobbler-test

Comment (Conan-Kudo) [reply cobbler.spec]:
```
[  105s] /bin/cp: cannot stat '/home/abuild/debbuild/BUILDROOT/cobbler-1579167507.b385f57f-1.1.amd64/etc/apache2/conf-available/cobbler.conf': No such file or directory
```

Comment (hbokh) [reply cobbler.spec]: @Conan-Kudo Are you also building on Debian Buster? because when I do "it just works" :)
See #2238

File: setup.py [hunk 9]
```diff
@@ -484,7 +484,7 @@ def run(self):
         http_user = "wwwrun"
         httpd_service = "apache2.service"
         defaultpath = "/etc/sysconfig/"
-    elif os.path.exists("/etc/debian_version"):
+    elif distro.id() in ("debian", "ubuntu"):
         if os.path.exists("/etc/apache2/conf-available"):
```
Comment (Conan-Kudo) [inline setup.py]: We can remove the `os.path.exists` check and just always assume under debian-ish systems that `webconfg = "/etc/apache2/conf-available"`. There is no supported version of Debian/Ubuntu where that doesn't exist.

Comment (SchoolGuy) [reply setup.py]: Removed it.

File: cobbler.spec [hunk 10]
```diff
@@ -350,6 +357,7 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %license COPYING
 %doc AUTHORS.in README.md
 %config(noreplace) %{apache_webconfigdir}/cobbler_web.conf
+%config(noreplace) %{apache_webconfigdir}/cobbler.conf
```
Comment (Conan-Kudo) [inline cobbler.spec]: Why is this here? We already have it in the main cobbler package. Should we remove it from the main cobbler package?

Comment (SchoolGuy) [reply cobbler.spec]: Yes we indeed do have this in there but when I remove it the package build fails on deb based systems if I reproduced this correctly. In this case I think it won't hurt.

File: cobbler.spec [hunk 11]
```diff
@@ -324,7 +328,7 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %license COPYING
 %doc AUTHORS.in README.md
 %doc docs/developer-guide.rst docs/quickstart-guide.rst docs/installation-guide.rst
-%config(noreplace) %{_sysconfdir}/cobbler
+%config(noreplace) %{_sysconfdir}/cobbler/
```
[the remark this answers is not in the document]
Comment (SchoolGuy) [reply cobbler.spec]: There are not only files with the `.conf ` suffix there like `version` or `cheetah_macros`. I think although what you are suggestion might be better I think we are safer using my fix.

File: cobbler.spec (hunk 8 above)
Comment (SchoolGuy) [reply cobbler.spec]: Now it works apparently.

File: cobbler.spec (hunk 11 above)
[the remark this answers is not in the document]
Comment (Conan-Kudo) [reply cobbler.spec]: Debian does not allow Conffiles to be a directory path, so it needs to be enumerated as files. Just add `*` afterward, then.

**Comment by SchoolGuy** [comment]
Great now SLE15 fails because I doubled the `.conf` file... I will check again.

File: cobbler.spec (hunk 11 above)
[the remark this answers is not in the document]
Comment (SchoolGuy) [reply cobbler.spec]: Okay thanks.

File: config/bash/completion/cobbler [hunk 12]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (hbokh) [inline config/bash/completion/cobbler]: Please revert. On Debian `bash` is in `/usr/bin/`:
```
# which bash
/usr/bin/bash
```

File: cobbler.spec (hunk 6 above)
Comment (hbokh) [reply cobbler.spec]: I am not used yet to reviews on GH (we use RBCommons instead), so bear with me.
Since I can't select older lines, I write it here.
The RAND_SECRET is not working in the `sed` that follows, WHEN it contains an ampersand (`&`). This happens 98 out of 100 times. See #2238 
Ah. just add a diff instead for the other issues mentioned there:

```diff
# diff cobbler.spec-mine cobbler.spec
39c39
< %define apache_webconfigdir /etc/apache2/conf-available
---
> %define apache_webconfigdir /etc/apache2/conf.d
262,269c262
< mkdir -p %{buildroot}%{tftpboot_dir}/boot
< mkdir %{buildroot}%{tftpboot_dir}/etc
< mkdir %{buildroot}%{tftpboot_dir}/grub
< mkdir %{buildroot}%{tftpboot_dir}/images
< mkdir %{buildroot}%{tftpboot_dir}/images2
< mkdir %{buildroot}%{tftpboot_dir}/ppc
< mkdir %{buildroot}%{tftpboot_dir}/pxelinux.cfg
< mkdir %{buildroot}%{tftpboot_dir}/s390x
---
> mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,s390x}
326,327c319
< #RAND_SECRET=$(openssl rand -base64 40 | sed 's/\//\\\//g')
< RAND_SECRET=$(head /dev/urandom | tr -dc 'A-Za-z0-9!"#$%'\''()*+,-./:;<=>?@[\]^_`{|}~' | head -c 57 ; echo '')
---
> RAND_SECRET=$(openssl rand -base64 40 | sed 's/\//\\\//g')
```

File: config/bash/completion/cobbler (hunk 12 above)
Comment (SchoolGuy) [reply config/bash/completion/cobbler]: Ah now we got a problem... The buildservice doesn't like `env`. I will try to think of something to get around `env` and do it correctly on both sides...

Comment (SchoolGuy) [reply config/bash/completion/cobbler]: I fixed it via a patch which just gets applied on the debian based build.

**Comment by Conan-Kudo** [comment]
@SchoolGuy The patch isn't required. `#!/bin/bash` is a valid shebang in all circumstances.

**Comment by hbokh** [comment]
> @SchoolGuy The patch isn't required. `#!/bin/bash` is a valid shebang in all circumstances.

My bad. On my testing Debian Buster, both `/usr/bin/bash` and `/bin/bash` exist for some reason (hard linked).

**Comment by hbokh** [comment]
@SchoolGuy I think it is only fair to thank you for your hard work! I really appreciate that there is a DEB file available that's almost instantly usable! Vielen dank! 💐

That said: do you consider installing the build DEBs on a host somehow?
I ask because maybe than you can check if the SECRET_KEY in `/cobbler/web/settings.py` is valid (I doubt it).
And if the directories in `/var/lib/tftpboot` are not joined together into one.

**Comment by SchoolGuy** [comment]
@Conan-Kudo @hbokh I will then remove the patch but will leave the structure for patches which may be needed in the future. And of course document how it is done to add new patches.

**Comment by SchoolGuy** [comment]
> That said: do you consider installing the build DEBs on a host somehow?

Nope I have not set something up for that. We have a lot on the backlog so this is at the moment not the highest prio but it is definitely on the bucket list.

> I ask because maybe than you can check if the SECRET_KEY in `/cobbler/web/settings.py` is valid (I doubt it).

I never checked but I will see what I can do.

> And if the directories in `/var/lib/tftpboot` are not joined together into one.

Sorry I do not understand. What do you mean by joined together into one?

**Comment by hbokh** [comment]
> What do you mean by joined together into one?

This:
```
ls -al /var/lib/tftpboot/
total 12
drwxr-xr-x  3 root root 4096 Jan 15 13:21 .
drwxr-xr-x 40 root root 4096 Jan 15 09:26 ..
drwxr-xr-x  2 root root 4096 Jan 15 10:55 {boot,etc,grub,images{,2},ppc,pxelinux.cfg,s390x}
```

**Comment by SchoolGuy** [comment]
Ah that sounds like something for @loreson

**Comment by SchoolGuy** [comment]
Rebased

File: cobbler.spec [hunk 13]
```diff
@@ -122,6 +122,7 @@ Group:          Development/System
 
 License:        GPL-2.0-or-later
 Source0:        https://github.com/cobbler/cobbler/archive/v%{version}/%{name}-%{version}.tar.gz
+Patch0:		0001-cobbler-completition-adjust-bash-shebang.patch
```
Comment (Conan-Kudo) [inline cobbler.spec]: Drop the patch.

File: cobbler.spec [hunk 14]
```diff
@@ -237,14 +241,18 @@ http://server/cobbler_web to configure the install server.
 
 
 %prep
-%autosetup -p1
+%setup
+
+# Apply debian patches
+%if %{_vendor} == "debbuild"
+%patch -P 0  -p1
+%endif
```
Comment (Conan-Kudo) [inline cobbler.spec]: Drop this.

File: cobbler.spec [hunk 15]
```diff
@@ -348,6 +363,9 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %{python3_sitelib}/cobbler/
 %{python3_sitelib}/cobbler-*
 %{_unitdir}/cobblerd.service
+%if 0%{?suse_version}
+%{_sbindir}/rccobblerd
+%endif
 %{tftpboot_dir}/*
 %{apache_dir}/cobbler
 %config(noreplace) %{_sharedstatedir}/cobbler
```
Comment (Conan-Kudo) [inline cobbler.spec]: This should probably not be `%config(noreplace)` I probably goofed here... :/

File: patches/debian/0001-cobbler-completition-adjust-bash-shebang.patch [hunk 16]
```diff
@@ -0,0 +1,10 @@
+diff --git a/config/bash/completion/cobbler b/config/bash/completion/cobbler
```
Comment (Conan-Kudo) [inline patches/debian/0001-cobbler-completition-adjust-bash-shebang.patch]: Kill this patch.

**Comment by SchoolGuy** [comment]
I am currently under way of documenting the patch process. After I have finished I will follow @Conan-Kudo 's suggestions and then merge the PR. If anyone objects please speak up.

**Comment by SchoolGuy** [comment]
WIP entry: https://github.com/cobbler/cobbler/wiki/Packaging-Cobbler

File: cobbler.spec (hunk 13 above)
Comment (SchoolGuy) [reply cobbler.spec]: Done

File: cobbler.spec (hunk 14 above)
Comment (SchoolGuy) [reply cobbler.spec]: Done

File: patches/debian/0001-cobbler-completition-adjust-bash-shebang.patch (hunk 16 above)
Comment (SchoolGuy) [reply patches/debian/0001-cobbler-completition-adjust-bash-shebang.patch]: Done

File: cobbler.spec (hunk 15 above)
Comment (SchoolGuy) [reply cobbler.spec]: Done

**Comment by SchoolGuy** [comment]
All distros built successfully so we merge this now. The process of who to submit a patch is in above mentioned comment done.
